client/pages/Login/ResetPassword/ResetPassword.tsx

```
@@ -1,37 +1,115 @@
-import '../login.scss'
 import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
 import { useTranslation } from 'react-i18next'
+
+import { useAppDispatch } from '@client/store'
+import { LoginActions } from '@client/store/login'
+import { LoginValidator } from '@client/pages/Login/utils/LoginValidator'
+
 import { Urls } from '@client/utils'
 import { Objects } from '@core/utils'
 
+import { BasePaths } from '@client/basePaths'
+
 const ResetPassword: React.FC = () => {
+  const dispatch = useAppDispatch()
   const { i18n } = useTranslation()
+  const history = useHistory()
+
+  const resetPasswordUuid = Urls.getRequestParam('resetPasswordUuid')
   const paramEmail = Urls.getRequestParam('email')
-  const isParamEmail = !Objects.isEmpty(paramEmail)
+
   const [email, setEmail] = useState<string>(paramEmail || '')
+  const [password, setPassword] = useState<string>('')
+  const [password2, setPassword2] = useState<string>('')
+  const [errors, setErrors] = useState<Record<string, string>>({})
 
-  const onChangePassword = () => {
-    console.log('not implemented')
+  const onResetPassword = async () => {
+    const fieldErrors = LoginValidator.resetPasswordValidate(email)
+    setErrors(fieldErrors)
+
+    if (!fieldErrors.isError) {
+      try {
+        await dispatch(LoginActions.createResetPassword(email)).unwrap()
+        history.push(BasePaths.Root())
+      } catch (rejectedValueOrSerializedError) {
+        // console.log(rejectedValueOrSerializedError)
```

Can we omit empty catch?
Or will the #1100 handle this?